particles/Experimental/Person_Msg_Demo/people.json

```diff
@@ -0,0 +1,5 @@
+[
+  {"name": "Jill", "age": 70},
```

Comment:
These people seem to be missing their `id`s? (from the schema above). Is this intended?

Reply:
Possibly this file is unused?